lib/ansible/modules/system/lvg.py

```diff
@@ -3,7 +3,8 @@
 
 # (c) 2013, Alexander Bulimov <lazywolf0@gmail.com>
 # based on lvol module by Jeroen Hoekx <jeroen.hoekx@dsquare.be>
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
This is unrelated to the rest of the pr. And this should be kept consistent with other code.